src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -175,9 +175,18 @@ define([
         },
 
         bindCustomEvents: function() {
+            // set this.saved{
+            this.sandbox.on('sulu.header.toolbar.item.enable',function (param) {
+                if (param === 'save') {
+                    this.saved = false;
```

Comment:
That feels like a bad connection... Hasn't the `this.saved` variable be set correctly before? I would prefer if the toolbar is dependent on the `this.saved` variable, and not the other way round.